specification/2.0/README.md

```diff
@@ -836,34 +836,30 @@ A skin is instanced within a node using a combination of the node's `mesh` and `
 
 ## Texture Data
 
-**TODO: describe separation of concerns: textures / images / samplers**
+glTF separates texture access into three distinct types of objects: Textures, Images, and Samplers.
 
 ### Textures
 
-**TODO: update to latest revisions**
-
-All textures are stored in the asset's `textures` array. A texture is defined by an image file, denoted by the `source` property; `format` and `internalFormat` specifiers, corresponding to the GL texture format types; a `target` type for the sampler; a sampler identifier (`sampler`), and a `type` property defining the internal data format. Refer to the GL definition of `texImage2D()` for more details.
+All textures are stored in the asset's `textures` array. A texture is defined by an image resource, denoted by the `source` property and a sampler index (`sampler`).
 
 ```json
 {
     "textures": [
         {
-            "format": 6408,
-            "internalFormat": 6408,
             "sampler": 0,
-            "source": 2,
-            "target": 3553,
-            "type": 5121
+            "source": 2
         }
     ]
 }
 ```
 
+> **Implementation Note** glTF 2.0 supports only 2D textures.
+
 ### Images
 
-**TODO: update to latest revisions**
+Images referred to by textures are stored in the `images` array of the asset. 
 
-Images referred to by textures are stored in the `images` array of the asset. Each image contains a URI to an external file (or a reference to a bufferView) in one of the supported images formats. Image data may also be stored within the glTF file as base64-encoded data and referenced via data URI. For example:
+Each image contains a URI to an external file (or a reference to a bufferView) in one of the supported images formats. Image data may also be stored within the glTF file as base64-encoded data and referenced via data URI. For example:
```

Comment:
Perhaps reorganize this paragraph as:

Each image contains one of
* a URI to an external file in one of the supported images formats, or
* a URI with embedded base64-encoded data, or
* a reference to a `bufferView`.

The following example shows an image pointing to an external .png image file and another image referencing a `bufferView`.